test/correctness/simd_op_check.cpp

```diff
@@ -1485,7 +1485,7 @@ int main(int argc, char **argv) {
     }
 
     // Compile a runtime for this target, for use in the static test.
-    compile_standalone_runtime("simd_op_check_runtime.o", target);
+    compile_standalone_runtime(Outputs().object("simd_op_check_runtime"), target);
```

Comment:
I can change the public-facing API on this to (say)

   compile_standalone_runtime(object_name, target)
   compile_standalone_runtime_library(library_name, target)

The real use case I want to fix is the GenGen command line.